### PR TITLE
First question

### PR DESCRIPTION
**Pull request by DennisTarik**
added some functions and defined the answers

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JavaQuiz</title>
```
Comment (marwinburesch) [inline index.html:7]: I think the title is a little misleading since the quiz is probably about JavaScript right?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="quiz">
+    <h1 class="title">JavaQuiz</h1>
```
Comment (marwinburesch) [inline index.html:11]: Same here, I think you need a new name :)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    <p class="question">Bist du eine Mensch</p>
```
Comment (marwinburesch) [inline index.html]:
```suggestion
    <p class="question">Bist du eine Mensch?</p>
```

Comment (leonardpaulus) [inline index.html:12]:
```suggestion
    <p class="question">Bist du ein Mensch</p>
```

File: index.html [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (leonardpaulus) [inline index.html:2]:
```suggestion
<html lang="de">
```

**Review by marwinburesch — COMMENTED** [review]
:)

**Pull request merged**